src/components/Button/index.js

```diff
@@ -1,4 +1,5 @@
 import React, {PropTypes} from 'react'
+import 'tachyons-egghead'
```

Comment:
Why move this from `.storybook/config.js`? It's needed globally in all components.